site/sass/components/_external-community.scss

```diff
@@ -22,4 +25,15 @@
     max-height: initial;
     width: $logo-size;
   }
+
+  &__copy-wrapper {
+    color: var(--grey-850);
+    font-family: Helvetica, Arial, sans-serif;
+    line-height: 1.3;
+
+    strong {
```

Comment:
Except in our `type` definitions and where we don't have control over the output HTML, we should try and avoid styling tags directly whenever possible